lib/renderer/web-view/guest-view-internal.ts

```diff
@@ -66,18 +67,18 @@ const dispatchEvent = function (
 };
 
 export function registerEvents (webView: WebViewImpl, viewInstanceId: number) {
-  ipcRendererInternal.on(`ELECTRON_GUEST_VIEW_INTERNAL_DESTROY_GUEST-${viewInstanceId}`, function () {
+  ipcRendererInternal.on(`GUEST_VIEW_INTERNAL_DESTROY_GUEST-${viewInstanceId}`, function () {
```

Comment:
can we use the enum here?

Reply:
This seems like a potentially useful typescript feature coming in 4.1: https://devblogs.microsoft.com/typescript/announcing-typescript-4-1-beta/#template-literal-types

Reply:
forgot this one